setup.py

```diff
@@ -37,7 +37,7 @@
     "pytest-html==1.22.0",  # Newer versions require Python 3
     "pytest-cov==2.7.1",
     "pytest-helpers-namespace==2019.1.8",
```

Comment:
It may be worth checking whether it is possible to update the pytest plugins together with pytest itself

Reply:
Fixed